.github/workflows/publish_image_chart.yaml

```diff
@@ -59,40 +57,12 @@ jobs:
             container_registry: gcr.io/pipecd
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
-      - uses: actions/setup-go@v3
-        with:
-          go-version: ${{ env.GO_VERSION }}
-          cache: true
       - name: Determine version
```

Comment:
[ASK] Is this line still needed?

Reply:
@ponkio-o 
Sorry for the wrong selection.
I'm asking about only this line.
```yaml
  - name: Determine version
```

Reply:
Sorry, the below line is needed. I was revert.
```yaml
run: echo "PIPECD_VERSION=$(git describe --tags --always --abbrev=7)" >> $GITHUB_ENV
```

https://github.com/pipe-cd/pipecd/pull/5420/commits/1fb552384dda94d7c459dbdddfaa74ab6a4e88ef